packages/datadog-plugin-next/test/index.spec.js

```diff
@@ -41,7 +41,8 @@ describe('Plugin', function () {
           server.stdout.on('data', chunk => process.stdout.write(chunk))
         })
 
-        after(async () => {
+        after(async function () {
+          this.timeout(5000)
```

Comment:
Why do we need a timeout now?
AFAIK the only thing that this PR do is check `pathname` parameter.